tests/WE2E/run_WE2E_tests.sh

```diff
@@ -998,13 +1012,11 @@ model_ver="we2e""
 #
 # Set OPSROOT.
 #
-    OPSROOT=${opsroot:-$( readlink -f "$HOMEdir/../nco_dirs" )}
-
     expt_config_str=${expt_config_str}"
 #
 # Set NCO mode OPSROOT
 #
-OPSROOT=\"${OPSROOT}\""
+OPSROOT=\"${opsroot}\""
```

Comment:
How does OPSROOT get set now?

Reply:
Both `expt_basedir` and `OPSROOT` have default values specified in `setup.py` so that they work in the case where we use `generate_FV3LAM_wflow.py` directly. Just wanted to be consistent with `expt_basedir`